web/src/pages/Upload.jsx

```diff
@@ -53,97 +68,105 @@ export const FileUploading = () => {
 				return false;
 			}
 			setFileList((prevList) => [...prevList, file]);
-			return true; // Allow the file to be added to the list
```

Comment:
Was there a reason for deleting this comment?

Reply:
which one?

Reply:
This one :)

`// Allow the file to be added to the list`

I noticed it was removed and wondered why 🤷